package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "devDependencies": {
+        "@percy/cli": "^1.0.0-beta.50"
```

Comment:
in case there is a change needed for below separate comment, please update this too: 1.0.0-beta.52